src/k8s/fa_configmaps.tf

```
@@ -21,3 +21,17 @@ resource "kubernetes_config_map" "fa-jvm" {
     TZ = "Europe/Rome"
   }
 }
+
+
+resource "kubernetes_config_map" "fa-ms-customer" {
+  metadata {
+    name      = "famscustomer"
+    namespace = kubernetes_namespace.fa.metadata[0].name
+  }
+
+  data = merge({
+    POSTGRES_SCHEMA = "fa_customer"
+    TZ = "Europe/Rome"
+  }, var.configmaps_facustomer_override)
```

```suggestion
  }, var.configmaps_facustomer)
```
only for take a standard naming convention